batch7crud-roll10/src/main/java/com/lftechnology/batch7crud/dao/StudentDao.java

```diff
@@ -0,0 +1,164 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.constant.EntityConstant;
+import com.lftechnology.batch7crud.constant.QueryConstant;
+import com.lftechnology.batch7crud.entity.Student;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.utils.DbUtils;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * A StudentDao class performs the CRUD operation in Student entity.
+ *
+ * @Author Binod Shrestha <binodshrestha@lftechnology.com>
+ * Created on 1/14/16
+ */
+public class StudentDao {
+  private static final Logger LOGGER = Logger.getLogger(StudentDao.class.getName());
+
+  public List<Student> fetch(Integer offset, Integer limit) throws DataException {
+    List<Student> studentList = new ArrayList<>();
+    ResultSet studentResult;
+
+    try (Connection conn = DbUtils.getConnection();
+            PreparedStatement ps = conn.prepareStatement(QueryConstant.FETCH_STUDENT_LIMIT_OFFSET)
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) "" is provided externally to the method and not sanitized before use. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2077)